### ISSUE TITLE
TinaFace: A lightweight backend for face detection

### ISSUE THREAD

**Issue by Raghucharan16** [serengil/deepface#1263]
### Description

As mentioned on Retinaface repo, Tinaface is light weight model which has better accuracy on wider {hard} dataset. and also it is efficient compared to retinaface. so could you add this to face detector backends?
thanks.
the results i talked about are mentioned [here](https://paperswithcode.com/sota/face-detection-on-wider-face-hard)
and pretrained model weights are found below:
[link](https://github.com/Media-Smart/vedadet/tree/main/configs/infer/tinaface)

**Comment by serengil** [comment]
its pre-trained weights are not available:

https://github.com/Media-Smart/vedadet/blob/main/configs/infer/tinaface/tinaface_r50_fpn_bn.py#L106

**Comment by Raghucharan16** [comment]
https://github.com/Media-Smart/flexinfer/tree/master/examples/object_detection
 My bad check out this, you will have both retinaface and tinaface model's weights
